src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -285,6 +285,23 @@
     <Compile Include="System\ExitCodeTests.Unix.cs" />
   </ItemGroup>
 
+  <!--
+    These tests are currently not R2R-Compatible, so we disable them whenever
+    we are using the TestReadyToRun flag.
+  -->
```

Comment:
Can we create a GitHub tracking issue and mention it in the comment?

Reply:
Will do.